lib/mixins/listen-on-root.js

```diff
@@ -1,15 +1,13 @@
+if (!Array.isArray) {
+    Array.isArray = function(arg) {
+        return Object.prototype.toString.call(arg) === "[object Array]"
+    }
+}
```

Comment:
You could just make a utility function `isArray(...)` that tried for the native `Array.isArray` first, and if not available then use the `Object.prototype.toString(...)` method.